app/services/proofing/proofer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Proofing
+  class Proofer
```

Comment:
While I like the idea of using interfaces, I think that we should discuss this with others on the Login program.